src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is running");
```

Comment:
The message in the `doWork` method should indicate what type of `Machine` is working. Consider specifying that it's an `Excavator` that is running.